docs/user/merging-pdfs.md

```diff
@@ -92,10 +93,11 @@ fields will be listed.
 
 ## reset_translation
 
-During cloning, if an object has been already cloned, it will not be cloned again, and a pointer
-to this previously cloned object is returned instead. Because of that, if you add/merge a page that has
-already been added, the same object will be added the second time. If you modify any of these two pages later,
-both pages can be modified independently.
+During cloning, if an object has been already cloned, it will not be cloned
```

Comment:
Do we really have any benefits from this?

Reply:
It only changes the source file (slightly better?) not the documentation webpage.

Reply:
I think we should avoid modifying the line length of source files without having any specific reason to reduce the general noise.